api/job.proto

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+syntax = "proto3";
+package openpitrix;
+
+// set go package name to pb
+option go_package = "pb";
```

Comment:
openpitrix_pb is better than pb, we can use pb as the import alias name: `import pb "path"`

and we can use `package name` to set origin package name.
`option go_package` most use to set package import path (imported by other proto file).

Reply:
No, it's not necessary.